teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.        # Add column named 'Target' for easier data visualization
Line #50.        test_pairs['Target'] = test_pairs['accession'].apply(lambda x: [i for i in targets_dict.keys() if targets_dict[i]==x][0])
</pre><p> </p><p>I suggest to make use of dictionaries' lookup capabilities directly (to avoid looping over a dictionary) with a small detour:</p><pre class="ql-syntax" spellcheck="false"># Add column named 'Target' for easier data visualization
# First, exchange keys (target) and values (UniProt) in targets_dict
_targets_dict_reversed = {uniprot: target for target, uniprot in targets_dict()}
# Second, look up target by UniProt accession
test_pairs['Target'] = test_pairs['accession'].apply(lambda x: _targets_dict_reversed[x])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>